README.md

```diff
@@ -1 +1,66 @@
+# Agorex - The Cooperative Decentralized Exchange
+[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/GVuj6UY)
+[![Join us on Reddit](https://img.shields.io/badge/reddit-Agorex-red.svg)](https://www.reddit.com/r/Agorex/)
+
+Our goal is to be the world’s first honest, community-run, decentralized exchange that leaves every trader smiling.
+
+- [Name](#name)
+- [Contribute](#contribute)
+- [Milestones](#milestones)
+- [Members](#members)
+- [Social Links](#social-links)
+
+## Name
+**Agorex, the Cooperative Decentralized Exchange**
+
+> So, what exactly does "Agorex" mean?
+
+Agorex is a combination of two words: _Agora_ & _Exchange_.
+
+An [_Agora_](https://en.wikipedia.org/wiki/Agora) was a _"public, open space used for assemblies and markets"_ in ancient Greece.
+
+As an added bonus _Agora_ also means _"Now"_ in Portuguese!
+
+We believe this name represents our core values of providing a **public**, **open**, **honest**, **community-run decentralized exchange**.
+
+
+## Contribute
+We are a group who have experience with decentralized exchanges. We want the community, like the technology, to also be decentralized. So, we are building a community-run decentralized cryptocurrency exchange that puts the community in charge. [Agorex Improvement Proposals (AIPs)](https://github.com/Agorex-io/Proposals) are the primary mechanisms for the community to propose new processes, raise issues, and collect community input on processes and issues. AIPs serve as the official documentation for the processes that the Board of Directors executes on behalf of the community. Additionally, we have set up a cooperative organization where contributors who want to make a more significant contribution can literally operate and manage the exchange on behalf of the community.
+
+### _How?_
+You can contribute by joining our [Discord server](https://discord.gg/GVuj6UY) and offering your ideas, insights, and advice on how to build a public, open, honest community around a decentralized exchange. You can contribute with your insights on all aspects of a community-run decentralized cryptocurrency exchange. This includes: technology, business/legal, economics, community building, documentation. And of course, we :heart: pull requests from designers and developers too!
+
+**If you're interested in becoming a part of this early stage of the coop, please drop by the #offers-to-help channel in our [Discord server](https://discord.gg/GVuj6UY) and share the following:**
+- Why you'd like to be a part of the Coop that will be formed.
+- What are your strengths?
+- What can you provide to the Coop?
+- What experience do you have?
+- What is your full name?
+- (optional) provide a LinkedIn profile
+- (optional) provide your GitHub profile 
+
+## Milestones
+An official list of milestones is currently pending [discussion](https://discord.gg/GVuj6UY). Below is a very high-level overview of what we seek to accomplish. 
```

Comment:
This should be improved.

Maybe a link to the master todo list would suffice for now?